controllers/comments.controller.js

```
@@ -0,0 +1,18 @@
+const { checkIfArticleExists } = require("../models/articles.model");
+const { addComment } = require("../models/comments.model");
+
+exports.postComment = (req, res, next) => {
+  const { username, body } = req.body;
+  const { article_id } = req.params;
+  const promises = [
+    checkIfArticleExists(article_id),
```

PSQL should throw an error if the article doesn't exist because the article_id is a foreign key and all foreign keys must be valid